monkey/monkey_island/cc/ui/src/services/configuration/config_schema.js

```diff
@@ -0,0 +1,63 @@
+import {customPBAConfigurationSchema} from './definitions/custom_pbas.js';
+import {pluginConfigurationSchema} from './definitions/plugins.js';
+import {propagationConfigurationSchema} from './definitions/propagation.js';
+import {bruteForceExploiters, vulnerabilityExploiters} from './definitions/exploiter_classes.js';
+
+export const SCHEMA = {
+  'title': 'Monkey',
+  'type': 'object',
+  'definitions': {
+    'brute_force_classes': bruteForceExploiters,
+    'vulnerability_classes': vulnerabilityExploiters
+  },
```

Comment:
Why are only these fields in `definitions`? Are we adding more later?
What's the difference between `definitions` and `properties`?

Reply:
I think we will add, fingerprinters, credential collectors and pbas to the definitions. Practically everything that is defined as plugin. 

Read this on the difference: https://cswr.github.io/JsonSchema/spec/definitions_references/